src/bin/combine.rs

```diff
@@ -91,8 +113,8 @@ fn main() {
                 .iter()
                 .map(|b| format!("{:02x}", b))
                 .collect::<String>();
-            eprintln!("Warning: Invalid utf-8 text, some symbols may be lost!");
-            eprintln!("Note: The hex representation of the secret is '{}'.", hex);
+            info!("Warning: Invalid utf-8 text, some symbols may be lost!");
+            debug!("Note: The hex representation of the secret is '{}'.", hex);
```

Comment:
I think `info!` may not be explicit enough to the user. Maybe change this to `warn!` and `info!`?